media-picker/build.gradle

```diff
@@ -18,15 +18,21 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    repositories {
+        jcenter()
+        maven { url 'https://jitpack.io' }
+    }
 }
 
 dependencies {
 
-    //noinspection GradleDynamicVersion || File provider compatibility.
-    compile 'com.android.support:support-core-utils:25.+'
+    // File provider compatibility.
+    compile 'com.android.support:support-core-utils:25.3.1'
 
-    //noinspection GradleDynamicVersion || File cropping utility.
-    compile 'me.villani.lorenzo.android:android-cropimage:1.+'
+    // File cropping utility.
+    compile 'com.github.yalantis:ucrop:2.2.1'
+    compile 'com.github.yalantis:ucrop:2.2.1-native'
```

Comment:
An additional 1.5MB seems like a steep price to pay for a small bump in image quality.

AFAIK most upstream consumers (Discord included) are going to already be compressing to a JPEG so not sure if we should be forcing it at this level, perhaps omit it and add a note in the readme that consumers can choose to include this explicitly if they wish.